frontend/src/pages/Home.jsx

```diff
@@ -1,14 +1,75 @@
 import styled from '@emotion/styled';
-import { Box, Button, TextField, Modal, Typography, Grid, InputLabel, MenuItem, FormControl, Select } from '@mui/material';
+import { Box, Button, TextField, Modal, Typography, Grid, InputLabel, MenuItem, FormControl, Select, Divider } from '@mui/material';
 import { Link } from 'react-router-dom';
 import * as React from 'react';
 import { useState, useEffect } from 'react';
 
+const StyledBackground = {
+  backgroundImage: 'url("/images/dji-air-4.png")',
+  backgroundSize: 'cover',
+  backgroundPosition: 'center',
+  backgroundAttachment: 'fixed',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  width: '100vw',
+  height: '100vh',
+};
+
+const StyledBox = {
+  background: 'linear-gradient(to bottom, #211401 14%, #3D250199 66%, #3C260350 100%)',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  textAlign: 'center',
+  width: '30vw',
+  height: '100vh',
+  transform: 'translateX(-93%)',
+};
+
+const StyledHiddenBox = {
+  background: 'linear-gradient(to bottom, #211401 24%, #3C260358 100%)',
+  display:'flex',
+  flexDirection:'column',
+  alignItems: 'left',
+  width: '62.9vw',
+  height: '20vh',
+  position: 'absolute',
+  top: '80vh',
+  right: '0vw',
+  visibility: 'hidden',
+  padding: '10px',
+  color: 'white',
+};
+
+const StyledDivider = {
+  width: '80%',
+  height: '4px',
+  backgroundColor: '#DEE2E6',
+  marginTop: '15px',
+  alignItems: 'center',
+};
+
 const StyledButton = styled(Button)`
   align-items: center;
   justify-content: center;
-  border-radius: 10px;
-  height: 50px;
+  border-radius: 5px;
+  height: 70px;
+  width: 370px;
+  color: 'white';
+  border: 2px solid #CC7E09;
+  background-color: transparent;
+  font-size: 38px;
+
+  &:hover {
+    background-color: #CC7E09;
+  }
+  
+  &:active{
+    background-color: #CC7E09;
+    border: 2px solid #CC7E09;
+  }
```

Comment:
@Hoshi-Okami move the above styles to styles folder, keeping only state and state updates in this component.